js/router.js

```diff
@@ -8,26 +8,36 @@ var ipcRenderer = require('ipc-renderer'),
     userPageView = require('./views/userPageVw'),
     donateView = require('./views/donateVw'),
     settingsView = require('./views/settingsVw'),
-    transactionsView = require('./views/transactionsVw');
+    transactionsView = require('./views/transactionsVw'),
+    PageConnectModal = require('./views/pageConnectModal');
 
 module.exports = Backbone.Router.extend({
-  routes: {
-    "": "index",
-    "home": "home",
-    "home/:state(/:searchText)": "home",
-    "myPage": "userPage",
-    "userPage": "userPage",
-    "userPage/:userID(/:state)(/:itemHash)(/:skipNSFWmodal)": "userPage",
-    "transactions": "transactions",
-    "transactions/:state": "transactions",
-    "settings": "settings",
-    "settings/:state": "settings",
-    "about": "about",
-    "support": "donate"
-  },  
-
   initialize: function(options){
+    var routes;
+
     this.options = options || {};
+
+    routes = [
```

Comment:
I moved the routes here so that we could apply them via the route method which gives the additional benefit of being able to use regex's in the route string.